### PR TITLE
Task/refactor

### PR DESCRIPTION
**Pull request by kenjones-cisco**

### CODE REVIEW

**Comment by kenjones-cisco** [comment]
Other than the etcd changes; these are the last of the changes I had made. Some came from kubernetes/contrib/ansible, and others I did for consistency.

File: group_vars/all.yml [hunk 1]
```diff
@@ -24,7 +24,17 @@ kube_service_addresses: 10.254.0.0/16
 # Flannel internal network (optional). When flannel is used, it will assign IP
 # addresses from this range to individual pods.
 # This network must be unused in your network infrastructure!
-kube_overlay_network: 4.0.0.0/16
```
Comment (altvnk) [inline group_vars/all.yml:27]: Is't it reasonable to have more common option here? For example if we add another optional networking project we can easily reuse same variable.

Comment (kenjones-cisco) [inline group_vars/all.yml:27]: `overlay_network_subnet`
`overlay_network_prefix`
`overlay_network_host_prefix`

Comment (altvnk) [inline group_vars/all.yml:27]: Looks good!

File: roles/flannel/tasks/main.yml [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
     dest: /tmp/flannel.json
-  run_once: true
   tags:
     - flannel
 
 - name: configure flannel in etcd
   run_once: true
   sudo: yes
-  command: 'curl -L http://{{ groups[etcd_group_name][0] }}:4001/v2/keys/coreos.com/network/config -XPUT --data-urlencode value@/tmp/flannel.json'
+  command: 'curl -L http://{{ groups[etcd_group_name][0] }}:4001/v2/keys/{{ dns_domain }}/network/config -XPUT --data-urlencode value@/tmp/flannel.json'
```
Comment (altvnk) [inline roles/flannel/tasks/main.yml:14]: Just wondering, should we start using here `etcdctl` as in contrib/ansible?

File: group_vars/all.yml (hunk 1 above)
Comment (kenjones-cisco) [inline group_vars/all.yml:27]: Cool, I'll make the changes!

**Comment by altvnk** [comment]
Looks awesome, I'm ready to merge. Waiting for changes in variables block.

**Comment by kenjones-cisco** [comment]
Variable name changes completed!

File: roles/flannel/tasks/main.yml (hunk 2 above)
Comment (kenjones-cisco) [inline roles/flannel/tasks/main.yml:14]: I think so, maybe take care of that as part of the etcd changes in general?

Comment (altvnk) [inline roles/flannel/tasks/main.yml:14]: Yep make sense.

**Pull request merged**